tutorials/roostats/rs401c_FeldmanCousins.py

```diff
@@ -0,0 +1,95 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook
+## Produces an interval on the mean signal in a number counting experiment with known background using the
+## Feldman-Cousins technique.
+##
+## Using the RooStats FeldmanCousins tool with 200 bins
+## it takes 1 min and the interval is [0.2625, 10.6125]
+## with a step size of 0.075.
+## The interval in Feldman & Cousins's original paper is [.29, 10.81] Phys.Rev.D57:3873-3889,1998.
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \date July 2022
+## \authors Artem Busorgin, Kyle Cranmer (C++ version)
+
+import ROOT
+
+# to time the macro... about 30 s
+t = ROOT.TStopwatch()
+t.Start()
+
+# make a simple model
+x = ROOT.RooRealVar("x", "", 1, 0, 50)
+mu = ROOT.RooRealVar("mu", "", 2.5, 0, 15)  # with a limit on mu>=0
+b = ROOT.RooConstVar("b", "", 3.0)
+mean = ROOT.RooAddition("mean", "", [mu, b])
+pois = ROOT.RooPoisson("pois", "", x, mean)
+parameters = {mu}
+
+# create a toy dataset
+data = pois.generate({x}, 1)
+data.Print("v")
+
+dataCanvas = ROOT.TCanvas("dataCanvas")
+frame = x.frame()
+data.plotOn(frame)
+frame.Draw()
+dataCanvas.Update()
+
+w = ROOT.RooWorkspace()
+modelConfig = ROOT.RooStats.ModelConfig("poissonProblem", w)
+modelConfig.SetPdf(pois)
+modelConfig.SetParametersOfInterest(parameters)
+modelConfig.SetObservables({x})
+w.Print()
+
+# show use of Feldman-Cousins
+fc = ROOT.RooStats.FeldmanCousins(data, modelConfig)
+fc.SetTestSize(0.05)  # set size of test
+fc.UseAdaptiveSampling(True)
+fc.FluctuateNumDataEntries(False)  # number counting analysis: dataset always has 1 entry with N events observed
+fc.SetNBins(100)  # number of points to test per parameter
+
+# use the Feldman-Cousins tool
+interval = fc.GetInterval()
+
+# make a canvas for plots
+intervalCanvas = ROOT.TCanvas("intervalCanvas")
+
+print("is this point in the interval? ", interval.IsInInterval(parameters))
+print("interval is [{}, {}]".format(interval.LowerLimit(mu), interval.UpperLimit(mu)))
+
+# using 200 bins it takes 1 min and the answer is
+# interval is [0.2625, 10.6125] with a step size of .075
+# The interval in Feldman & Cousins's original paper is [.29, 10.81]
+# Phys.Rev.D57:3873-3889,1998.
+
+# No dedicated plotting class yet, so do it by hand:
+parameterScan = fc.GetPointsToScan()
+hist = parameterScan.createHistogram("mu", 30)
```

Comment:
@guitargeek Why is it RooFit.Binning? I'm not sure where you are seeing that in the C++ version.
There is a rs401**d**_FeldmanCousins.C tutorial which uses the Binning, are you looking at that?

In rs401c_FeldmanCousins.C we have
`TH1F *hist = (TH1F *)parameterScan->createHistogram("mu", 30);`


When I try using `ROOT.RooFit.Binning(30)` getting an error message because there doesn't exist a two parameter overload of createHistogram(). 

> TypeError: none of the 3 overloaded methods succeeded. Full details:
  TH1* RooAbsData::createHistogram(const char* name, const RooAbsRealLValue& xvar, const RooCmdArg& arg1 = RooCmdArg::none(), const RooCmdArg& arg2 = RooCmdArg::none(), const RooCmdArg& arg3 = RooCmdArg::none(), const RooCmdArg& arg4 = RooCmdArg::none(), const RooCmdArg& arg5 = RooCmdArg::none(), const RooCmdArg& arg6 = RooCmdArg::none(), const RooCmdArg& arg7 = RooCmdArg::none(), const RooCmdArg& arg8 = RooCmdArg::none()) =>
    TypeError: could not convert argument 2
  TH1* RooAbsData::createHistogram(const char* name, const RooAbsRealLValue& xvar, const RooLinkedList& argList) =>
    TypeError: takes at least 3 arguments (2 given)
  TH1* RooAbsData::createHistogram(const char* varNameList, int xbins = 0, int ybins = 0, int zbins = 0) =>
    TypeError: could not convert argument 2 (int/long conversion expects an integer object)

Reply:
I also see it here in `rs401c_FeldmanCousins.C`:
https://github.com/root-project/root/blob/master/tutorials/roostats/rs401c_FeldmanCousins.C#L104

Probably you are not looking at the newest version from the `master` banch? You are right, in the 6.26 release it's not there yet, but in the upcoming release we want to encourage more explicit code where users pass the `Binning` argument. It is also supported in Python with the ROOT master, so the crash you report won't be there.

I think it's not worth it for you to compile ROOT `master` just that you can verify that the Python tutorial still works with my suggested change. Can you please just apply it, and then the CI will tell if it works?

Thank you so much for all your work!

Reply:
Sure, committed the change to the PR.

I see it now, you are right I was on the latest release, not master.